### PR TITLE
🔥[WIP][NOT READY FOR REVIEW]🐍[EXPERIMENT] 🐉 Use Ubuntu LTS as master image by default 🔥

### PR DESCRIPTION
**Pull request by dims**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: cluster/gce/gci/configure.sh [hunk 1]
```diff
@@ -393,6 +401,59 @@ function load-docker-images {
...
+    return
+  fi
+  # Install Docker deps, some of these are already installed in the image but
+  # that's fine since they won't re-install and we can reuse the code below
+  # for another image someday.
+  apt-get update
+  apt-get install -y --no-install-recommends \
+    apt-transport-https \
+    ca-certificates \
+    socat \
```
Comment (aojea) [inline cluster/gce/gci/configure.sh]: this won't work if we don't install docker, I imagine that this is only for testing but is easy to forget :smile:

**Comment by dims** [comment]
node-e2e had a flake - https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/87654/pull-kubernetes-node-e2e/1225265169563652096/

/test pull-kubernetes-node-e2e

**Comment by dims** [comment]
3 flakes and the cross timed out. No bazel related interrupt/timeout/ooms. the PR for bazel shutdown is holding up!

**Comment by dims** [comment]
This PR served us well. adios my friend. All the stuff here have gotten into k/k and k/test-infra resulting in:

https://testgrid.k8s.io/sig-release-master-informing#gce-ubuntu-master-default
https://testgrid.k8s.io/sig-release-master-informing#gce-ubuntu-master-containerd

/close

**Pull request closed without merging**